actors/builtin/miner/miner_actor.go

```diff
@@ -419,10 +418,10 @@ func (a Actor) ProveCommitSector(rt Runtime, params *ProveCommitSectorParams) *a
 
 	msd, ok := MaxSealDuration[precommit.Info.RegisteredProof]
 	if !ok {
-		rt.Abortf(exitcode.ErrIllegalState, "No max seal duration for proof type: %d", precommit.Info.RegisteredProof)
+		rt.Abortf(exitcode.ErrIllegalState, "no max seal duration for proof type: %d", precommit.Info.RegisteredProof)
 	}
 	if rt.CurrEpoch() > precommit.PreCommitEpoch+msd {
```

Comment:
nit: consider extracting `precommit.PreCommitEpoch+msd` to a variable and using it here and in the abort below.